### PR TITLE
Adding missing Celo ecosystem projects

### PR DESCRIPTION
**Pull request by viral-sangani**
This PR includes GitHub repositories built on celo. 
These repositories are part of the [Celo camp](https://www.celocamp.com/) and [Startup pathway](https://www.startup-pathway.com/).

### CODE REVIEW

File: data/ecosystems/c/celo.toml [hunk 1]
```diff
@@ -7961,3 +7961,82 @@ url = "https://github.com/koshikraj/safe2link"
...
+url = "https://github.com/walliDprotocol/celo-demo"
+
+[[repo]]
+url = "https://github.com/simbiotica-finance/self-reporting"
+
+[[repo]]
+url = "https://github.com/clixpesa/clixpesa-mvp"
+
+[[repo]]
+url = "https://github.com/staxeio/staxe-contracts/"
```
Comment (eherrerosj) [inline data/ecosystems/c/celo.toml:8006]: this one looks like polygon / ethereum only (?)

File: data/ecosystems/c/celo.toml [hunk 2]
```diff
@@ -7961,3 +7961,82 @@ url = "https://github.com/koshikraj/safe2link"
...
+url = "https://github.com/ecoland-world/eco_seeds"
+
+[[repo]]
+url = "https://github.com/joaovwfreire/ecoseeds"
+
+[[repo]]
+url = "https://github.com/ComiBlock/Yuki-Fiat-Network-Showcase/"
+
+[[repo]]
+url = "https://github.com/Nameer-Alkhayyat/axon-dapp"
```
Comment (eherrerosj) [inline data/ecosystems/c/celo.toml:8018]: this one looks like ethereum only (?)

**Review by eherrerosj — CHANGES_REQUESTED** [review]
left a couple comments

**Comment by viral-sangani** [comment]
Hey @eherrerosj Removed the two repos for now since I could not find the founders to verify.
Also removed few repos which are already existing.

**Review by eherrerosj — APPROVED** [review]
LGTM, thank you!!

**Pull request merged**